Makefile

```diff
@@ -20,11 +23,19 @@ pgdown:
 
 # building and testing
 build:
-	  go build -o ./build/nnmg$(TAG) nnmg.go
+		go build -o ./build/nnmg$(TAG) nnmg.go
```

Comment:
created very low pri ticket to normalize the spacing in this file. I think tabs in local are being transformed to 4 or 8 spaces in github, but only display as 2 or 4 spaces in vscode.